src/elements/ReviewCard.jsx

```diff
@@ -0,0 +1,11 @@
+export const ReviewCard = ({ title, owner, votes, date }) => {
+  return (
+    <div className="ReviewCard">
```

Comment:
As this is a list this component should return `li` tags and be wrapped in a `ul` or `ol`. There's semantic meaning to these tags so keep in good habits and put them in now.